pages/linux/netselect.md

```diff
@@ -0,0 +1,24 @@
+# netselect
+
+> Speed test for choosing a fast network server.
+> More information: <https://github.com/apenwarr/netselect> .
+
+- Choose the fastest server:
```

Comment:
Thanks for the clarification, and sorry it's taken me so long to get back to you. Perhaps this will read better?
```suggestion
- Choose the server with the lowest latency:
```

Reply:
This seems on point. 

Thanks!